### PR TITLE
Eth 596 subgraph fixeschanges

### PR DESCRIPTION
**Pull request by samigabor**

### CODE REVIEW

File: packages/network-contracts/contracts/OperatorTokenomics/Sponsorship.sol [hunk 1]
```diff
@@ -41,6 +41,7 @@ contract Sponsorship is Initializable, ERC2771ContextUpgradeable, IERC677Receive
 
     event StakeUpdate(address indexed operator, uint stakedWei, uint allocatedWei); // TODO change: allocatedWei -> earningsWei
     event SponsorshipUpdate(uint totalStakeWei, uint unallocatedWei, uint32 operatorCount, bool isRunning); // TODO: change uint32 -> uint, stake -> staked
+    event LeavePolicyParameter(uint minimumStakingPeriodSeconds);
```
Comment (samt1803) [inline packages/network-contracts/contracts/OperatorTokenomics/Sponsorship.sol:44]: move this parameter to the NewSponsorship event in the sponsorship factory

File: packages/network-subgraphs/src/sponsorship.ts [hunk 2]
```diff
@@ -23,6 +24,7 @@ export function handleStakeUpdated(event: StakeUpdate): void {
         stake = new Stake(stakeID)
         stake.sponsorship = sponsorshipAddress.toHexString()
         stake.operator = operatorAddress.toHexString()
+        stake.joinDate = event.block.timestamp
     }
     stake.date = event.block.timestamp
```
Comment (samt1803) [inline packages/network-subgraphs/src/sponsorship.ts:29]: delete .date

**Review by samt1803 — CHANGES_REQUESTED** [review]

File: packages/network-subgraphs/src/sponsorship.ts (hunk 2 above)
Comment (samigabor) [reply packages/network-subgraphs/src/sponsorship.ts]: Deleted

File: packages/network-contracts/contracts/OperatorTokenomics/Sponsorship.sol (hunk 1 above)
Comment (samigabor) [reply packages/network-contracts/contracts/OperatorTokenomics/Sponsorship.sol]: Moved and removed LeavePolicyParameter  event from Sopnsorship and handler from subgraph

**Review by samt1803 — APPROVED** [review]
looks good!

**Pull request merged**